src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -716,8 +716,17 @@ def test_run_app_bulk_from_gui_cell(self, auth, c):
     def test_run_app_bulk_dry_run(self, auth, c):
         mock_comm = MagicMock()
         c.return_value.send_comm_message = mock_comm
-        new_job = self.am.run_app_bulk(self.bulk_run_good_inputs, dry_run=True)
-        self.assertIsInstance(new_job, dict)
+        dry_run_results = self.am.run_app_bulk(self.bulk_run_good_inputs, dry_run=True)
```

Comment:
Check to see that only the right keys are present. There's already similar tests for the `run_app` function that include looking for the `wsid` key, so I just left those as-is.